src/compiler/scanner.ts

```diff
@@ -75,6 +75,8 @@ export interface Scanner {
     reScanInvalidIdentifier(): SyntaxKind;
     scanJsxToken(): JsxTokenSyntaxKind;
     scanJsDocToken(): JSDocSyntaxKind;
+    /** @internal */
+    scanJSDocCommentTextToken(inBackticks: boolean): JSDocSyntaxKind | SyntaxKind.JSDocCommentTextToken;

```

Comment:
If this is only internal because the new token is, we can probably flip it back pending the other thread.

Reply:
Flip what back?

Reply:
This to not being internal

Reply:
No, my understanding is that it's not a rescan - it's a specific type of scan which is the only one that can produce `JSDocCommentTextToken`s (which for all intents and purposes is just an internal optimization). Maybe @sandersn can verify.

Reply:
Yeah, I think I misunderstood. You can only generate this token by explicitly calling `scanJSDocCommentTextToken`. I was confused because the `JSDocCommentTextToken` option comes first in the switch case in the parser even though it isn't something you'll hit until you set `SavingComments` somewhere else.